frontend/apps/reader/modules/readersearch.lua

```diff
@@ -162,22 +170,18 @@ function ReaderSearch:onShowSearchDialog(text, direction)
             {
                 {
                     text = from_start_text,
-                    vsync = true,
```

Comment:
You still need to remove these @NiLuJe babies , now that you switched to Notifications ?

Reply:
I saw that @NiLuJe removed them when possible.
I played with this menu without `vsync` (flash_ui enabled) and faced no problems.

Reply:
You need a very specific range of devices to see the difference, and they're *definitely* needed, with no side-effects and unaffected devices, so, yep: please leave them be ;).

Reply:
Excuse me, reverted.